docs/decisions/0002-footer-customizations.rst

```diff
@@ -0,0 +1,138 @@
+customization option to be configurable through the `MFE's config mechanism`_.
+
+For styles, to allow a great degree of freedom in customization, we will allow,
+where appropriate, to either supply a list of CSS classes which will be applied
+to a particular component, or a list of CSS attributes and values.
+
+.. _MFE's config mechanism: https://github.com/openedx/frontend-platform/blob/master/src/config.js
+
+Consequences
+************
+
+.. This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+
+Positive
+========
+
+* Styles and additional content (i.e. all the customizations to the component)
+  can be configured in the same place (or at least using the same API).
+* Can set defaults at build time, and overwrite them at runtime.
+* Have the option to do at build time.
+* Can customize for each MFE independently.
+* Using ``eox-tenant`` can customize per tenant (org, microsite).
+
+Negative
+========
+
+* At the time of writing, Tutor doesn't pass ``env.config.js`` or environmental
+  variables at build time, which means that anyone using tutor directly, or
+  tools that are built on top of it (e.g. Grove, cookiecutter-openedx-devops)
+  will have to use ``MFE_CONFIG`` Web API.
+* There exist branding mechanism and theming mechanism for styling MFEs, and
+  the styles are expected to go there. The consequence of the decision above is
+  that the styles will be in a separate place.
+* Developers/designers would have to edit CSS styles and classes in the
+  python/yaml/js files, which can be inconvenient.
+* Configuration keys through which the customizations are exposed can
+  potentially overlap, causing collisions. Hard to catch and debug.
+
+Rejected Alternatives
+*********************
+
+.. This section lists alternate options considered, described briefly, with pros and cons.
+
+React component props API
+=========================
+
+Instead of reading the configuration values in the component, we could be
+reading it somewhere else, and passing the values as props to the react
+component. We couldn't identify any pros to this approach, and the cons are the
+same as the cons from decoupling.
+
+Styling using CSS variables
+===========================
+
+Exposing styling customizations through the CSS variables would give a benefit
+of being able to set the customizations in the branding package, together with
+the rest of the styles. The cons however, are that it would be harder to
+customize it for different MFEs and tenants, because you would need different
+branding packages for each, and the customization options would be limited to
+changing only the CSS attributes we decide can be changed, because you can't
+set arbitrary styles through a CSS variable, only a value for a CSS attribute.
```

Comment:
Why would it be harder for different MFEs? Each MFE can specify its own root class which automatically allows you to use that to target that specific MFE. 

Additionally, you will always need different branding packages for each tenant since the styling is different, whether that branding package is delivered as a css file or variables in the config API is all that will change.